doc/classes/File.xml

```diff
@@ -275,6 +275,7 @@
 			</argument>
 			<description>
 				Opens a compressed file for reading or writing.
+				[b]Note:[/b] [method open_compressed] can only read files that were saved by Godot. It cannot read generic [code].gz[/code] or [code].zst[/code] files. See [url=https://github.com/godotengine/godot/issues/28999]GitHub issue #28999[/url] for a workaround.
```

Comment:
I'm not sure this should mention specific compression format extensions like `.gz` or `.zst`. There are hundreds of compression formats and extensions for compressed files. It should be sufficient to say that this method is only intended for use with files compressed by Godot with `open_compressed`, and not any third-party compression format.